### ISSUE TITLE
Quantity of artwork is not decreased after being bought

### ISSUE THREAD

**Issue by milanbargiel** [milanbargiel/kulturgenerator#114]
**Steps to reproduce**
- Buy artwork with paypal https://staging.kulturgenerator.de/testkunstwerk/test
- See that artwork quantity remains 1 even though an order object is created in the backend

**Comment by moritzpflueger** [comment]
### steps to debug
* go to Milan's server
* stop the pm2 process
* cd into cms folder
* and do npm start
* now you can see console statements in the terminal
* head to staging.kulturgenerator.org and buy test artwork

**Comment by moritzpflueger** [comment]
fixed with 8a5229041a028c16221f5bcb60f5ccb0c9e400aa

### for the archives:

* it's important to call strapis `update` service with an Id wrapped in an object:
```js
strapi.services.artwork.update({ id: orderedArtwork.id }, { ... }); // like this

strapi.services.artwork.update(orderedArtwork.id, { ... }); // not like this !!!
```
